### PR TITLE
42136 setChartProperty should accept numbers

### PR DESCRIPTION
**Pull request by jdrichards50**
Tweak the setChartProperty and setEngineProperty methods along with some cleanup.

Chart engine string is now a constant.

### CODE REVIEW

**Comment by rbruckheimer** [comment]
I see 3 instances of "stxx" and one of "CIQ.MobileBridge" hardcoded in the file, instead of using the variables.
One of the stxx instances does not have a period; I was wondering why your variables included the trailing period instead of having the period in each function.

**Comment by jdrichards50** [comment]
> I see 3 instances of "stxx" and one of "CIQ.MobileBridge" hardcoded in the file, instead of using the variables. One of the stxx instances does not have a period; I was wondering why your variables included the trailing period instead of having the period in each function.

Good catch, I missed those more detailed scripts.

File: Source/ChartIQScriptManager.swift [hunk 1]
```diff
@@ -456,12 +467,14 @@ internal class ChartIQScriptManager: ChartIQScriptManagerProtocol {
...
   internal func getScriptForSetStudyParameters(_ study: ChartIQStudy, parameters: [String: String]) -> String {
     var script = getStudyDescriptorScript(study.fullName) +
-      "var helper = new CIQ.Studies.DialogHelper({sd:selectedSd,stx:stxx}); " + "var isFound = false; " +
+      "var helper = new CIQ.Studies.DialogHelper({sd:selectedSd,stx:" + chartIQJsObject + "}); " + "var isFound = false; " +
       "var newInputParameters = {}; " + "var newOutputParameters = {}; " + "var newParameters = {}; "
     parameters.forEach { parameter in
       script += getUpdateStudyParametersScript(parameter.key, value: parameter.value)
     }
-    script += "helper.updateStudy({inputs:newInputParameters, outputs:newOutputParameters, parameters:newParameters}); console.log(JSON.stringify(newParameters)) "
+    script += "helper.updateStudy({inputs:newInputParameters, outputs:newOutputParameters, parameters:newParameters}); console.log(JSON.stringify(newParameters));"
```
Comment (rbruckheimer) [inline Source/ChartIQScriptManager.swift:475]: Do you need the console logging?

Comment (jdrichards50) [reply Source/ChartIQScriptManager.swift:475]: Probably not, users can add it back in if they want.

**Review by rbruckheimer — APPROVED** [review]

**Pull request merged**